libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F732xx.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env python
+'''
+these tables are generated from the STM32 datasheets for the
+STM32F732
+'''
+
+# additional build information for ChibiOS
+build = {
+    "CHIBIOS_STARTUP_MK"  : "os/common/startup/ARMCMx/compilers/GCC/mk/startup_stm32f7xx.mk",
+    "CHIBIOS_PLATFORM_MK" : "os/hal/ports/STM32/STM32F7xx/platform.mk"
+    }
+
+# MCU parameters
+mcu = {
+    # ram map, as list of (address, size-kb, flags)
+    # flags of 1 means DMA-capable
+    # flags of 2 means faster memory for CPU intensive work
+    'RAM_MAP' : [
+        (0x20000000,  64,  3), # DTCM
```

Comment:
You have a comment above saying DTCM DMA does not work in which case this option is wrong

Reply:
where is that comment? DMA is fine on DTCM for F732.